src/test/java/com/bravo/user/service/PaymentServiceTest.java

```diff
@@ -0,0 +1,84 @@
+package com.bravo.user.service;
+
+import com.bravo.user.App;
+import com.bravo.user.dao.model.Payment;
+import com.bravo.user.dao.model.mapper.ResourceMapper;
+import com.bravo.user.dao.repository.PaymentRepository;
+import com.bravo.user.model.dto.PaymentDto;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import java.util.List;
+import java.util.UUID;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.Mockito.*;
+
+
+@ContextConfiguration(classes = {App.class})
+@ExtendWith(SpringExtension.class)
+@SpringBootTest
+class PaymentServiceTest {
+
+    @Autowired
+    private PaymentService paymentService;
+
+    @MockBean
+    private ResourceMapper resourceMapper;
+
+    @MockBean
+    private PaymentRepository paymentRepository;
+
+    private List<PaymentDto> dtoPayments;
+
+    @BeforeEach
+    void setUp() {
+        final List<Integer> ids = IntStream
+                .range(1, 10)
+                .boxed()
+                .collect(Collectors.toList());
```

Comment:
Nitpicking here. Is this better than a for loop?

Reply:
I don't think you're nitpicking, and I'm glad you made me look more closely at this part of the code. My short answer is no, it's probably not better than a for loop - depending on what we actually do in the for loop I suppose.

Had I used a for loop I doubt I would have ended up with as much object creation as is happening in this code. The boxing of int primitives and even having the List<Integer> itself probably isn't the best for memory usage and execution time. On top of that, in the second stream operation creating the `paymentDtoList`, the call to `Integer.toString()` is passed the Integer object, so it gets unboxed again in that call. This could compound as the code base grows and more tests are added, since this is in the `@BeforeEach` setup.

So... I'd either go with a simple re-initialization and for loop like this:

```
paymentDtoList = new ArrayList<>();
for (int i = 0; i < 10; i++) {
    paymentDtoList.add(createPaymentDto(Integer.toString(i)));
}
```
        
Or at least a trimmed down stream operation like this:

```
paymentDtoList = IntStream
        .range(1, 10)
        .mapToObj(i -> createPaymentDto(Integer.toString(i)))
        .collect(Collectors.toList());
```
        
Being really honest, I saw that code in another test class in the project and I think my geek brain said "Oooh, I like streams"  and I used it without thinking critically enough about it.  This is one of the reasons that code reviews are so helpful!

Also, I'll nitpick on myself here... thinking through this also reminded me that the end parameter in `range(1, 10)` is exclusive, so you get a list of nine items, whereas my simple loop provides 10 items. It doesn't matter to these test results, but reading "1, 10" it is easy to mentally expect 10 items. We could use `rangeClosed(1, 10)` to get inclusive of the end param.

Thanks for the feedback and for questioning that code. Let me know if you'd like an update to the pull request to address the main concern with those stream operations.